cmd/create/accountroles/creators.go

```diff
@@ -38,7 +38,7 @@ func initCreator(r *rosa.Runtime, managedPolicies bool, classic bool, hostedCP b
 	}
 
 	// Classic ROSA unmanaged policies
-	if classic {
+	if classic || r.Creator.IsGovcloud {
```

Comment:
Is this check needed? Can we not force classic to be true if we are on gov cloud instead?